### ISSUE TITLE
Fails to resolve local dependency

### ISSUE THREAD

**Issue by wbrickner** [nicholasbishop/aws-build#4]
`$ aws-build al2 --rust-version nightly`

fails because my `Cargo.toml` contains

```toml
[dependencies]
my_library = { path = "../libs/my_library" }
```

How do I move this directory into the docker container so that it can be found during build?

Thank you!

**Comment by nicholasbishop** [comment]
Right now I don't think it's possible. I'll take a look at fixing that.

**Comment by wbrickner** [comment]
Thank you, I really appreciate your hard work to make rust a first class language for cloud development. This is really valuable and worthwhile.

It is a bit of a limitation, right now the obvious workaround is to just copy the crate into the directory, but it's a very wonky solution.

**Comment by nicholasbishop** [comment]
This should be fixed now. I've added a new option, `--code-root`, to specify the path that gets mounted into the container containing all the code. The project path must be somewhere within that code root.

I've published a `0.10.0` release for this, let me know if you run into any issues.

**Comment by wbrickner** [comment]
Awesome, that's a perfect solution! Thanks so much!
